sphinx/sphinx.gradle

```diff
@@ -8,12 +8,22 @@ description = 'Generates docs for the python libraries provided in Deephaven Cor
 
 evaluationDependsOn ':Integrations'
 
+tasks.register('copyPyClientWhl', Copy) {
+    dependsOn ':pyclient:buildPyClient'
```

Comment:
must have an `evaluationDependsOn ":pyclient"` if we do this, so that the task is sure to exist in time, otherwise there is a risk that pyclient could depend on us and just fail with NPEs when we build

Reply:
this dependsOn is now unnecessary, the next line implicitly sets up that dependency